acitoolkit/__init__.py

```diff
@@ -42,7 +42,7 @@
     OSPFInterfacePolicy, OSPFRouter, OutputTerminal, OutsideEPG,
     OutsideL2, OutsideL2EPG, OutsideL3, OutsideNetwork,
     PhysDomain, PortChannel, Search, Subnet, Taboo, Tenant, TunnelInterface,
-    VMM, VMMCredentials, VmmDomain, VMMvSwitchInfo, Tag
+    VMM, VMMCredentials, VmmDomain, VMMvSwitchInfo, Tag, _interface_from_dn
```

Comment:
Not sure why we need to add this _interface_from_dn here.....

Reply:
I included this based on this (unmerged) commit from Cara:
https://github.com/noironetworks/acitoolkit/pull/7/commits/79ace1863bba8af17d3c01fb0a00e706c7d36cc8, which is part of https://github.com/noironetworks/acitoolkit/pull/7. The comment in her commit message for this was "Python 3 is unable to access this method unless it's imported". I don't think we're using this method in acisession.py, but thought I'd include this fix just in case. If you prefer, I can remove it.

Reply:
Its ok, we can keep it then. Thx.